### DISCUSSION TITLE
Thalami should not shoot railguns at players

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#7776] [Ideas and suggestions]
currently, it is possible to depart from your ship to board into a thalamus only to get one tapped by a railgun shell. I think that the railgun should only be used to shoot at the main sub, while coilgun is for players. pretty simple, hope to see this change made

**Comment by r4d6** [comment]
Then just board the thalamus from the side where it can't shoot you with a railgun.

**Reply by ghost** [reply]
Exactly, you should not just swim at a thalamus head on, and railguns are a fairly large incentive to explore alternate routes.

**Reply by StruggleDuck** [reply]
its not so much for me or other more experienced players, but I do not believe that a new player who doesn't even know what a thalamus is will understand that they need to avoid this one gun or will be instantly cut in half. That's the main issue I have with this.

**Reply by r4d6** [reply]
Well those doesn't even know what a thalamus is will get hooked and brought almost next to the sub, and at this point his sub would block the thalamus' railgun no?

**Comment by StruggleDuck** [comment]
Not neccucarily.
